packages/generator/src/edmx-to-vdm/v4/action-import.ts

```diff
@@ -0,0 +1,65 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+import { ServiceNameFormatter } from '../../service-name-formatter';
+import { swaggerDefinitionForFunctionImport } from '../../swagger-parser/swagger-parser';
+import {
+  EdmxAction,
+  EdmxActionImport,
+  parseActionImport,
+  parseActions
+} from '../../edmx-parser/v4';
+import { ServiceMetadata } from '../../edmx-parser/edmx-file-reader';
+import { VdmActionImport, VdmComplexType, VdmEntity } from '../../vdm-types';
+import { parseActionImportReturnTypes } from '../common/action-function-return-types';
+import { stripNamespace } from '../edmx-to-vdm-util';
+import { transformActionImportBase } from '../common/action-import';
+
+export function findActionForActionImport(
```

Comment:
It seems that you could reuse the generic functionality here? I guess there should be something in common?

Reply:
I add a shared function.